### PR TITLE
Asa/test

### PR DESCRIPTION
**Pull request by kruai**
This started as a test branch, but started to contain actual development. I'm merging in our progress so that we can continue work off of the master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,61 +2,80 @@ Insights Tag Service
 ===========================================
 
 Dev Setup
+
 --------------------
+
 1. Install dependencies
-```
+
```
Comment (tahmidefaz) [inline README.md]: Not sure if it's just me, but the formatting kind of seems off

Comment (kruai) [reply README.md]: Which part? I added the newline because it's required by markup lint.

Comment (tahmidefaz) [reply README.md]: The whole README :sweat_smile: 
image

Comment (kruai) [reply README.md]: Ah, thanks for the screenshot. It looks weird because the section headers aren't being formatted the same way now. I'll throw in a fix

Comment (tahmidefaz) [reply README.md]: Thanks! Looks good!

**Pull request merged**